<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zoomPanForm</class>
 <widget class="QWidget" name="zoomPanForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QToolButton" name="zoomOutButton">
       <property name="text">
        <string>-</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="zoomSlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="sliderPosition">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomInButton">
       <property name="text">
        <string>+</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="1">
      <widget class="QToolButton" name="panUpButton">
       <property name="text">
        <string>N</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="panLeftButton">
       <property name="text">
        <string>W</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="zoomAllButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="panRightButton">
       <property name="text">
        <string>E</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="panDownButton">
       <property name="text">
        <string>S</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="zoomOutAction">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="panUpAction">
   <property name="text">
    <string>Pan Up</string>
   </property>
  </action>
  <action name="panDownAction">
   <property name="text">
    <string>Pan Down</string>
   </property>
  </action>
  <action name="panLeftAction">
   <property name="text">
    <string>Pan Left</string>
   </property>
  </action>
  <action name="panRightAction">
   <property name="text">
    <string>Pan Right</string>
   </property>
  </action>
  <action name="zoomAllAction">
   <property name="text">
    <string>Zoom Extents</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
